internal/db/schema/manager.go

```diff
@@ -127,8 +156,9 @@ func (b *Manager) ExclusiveUnlock(ctx context.Context) error {
 }
 
 // RollForward updates the database schema to match the latest version known by
-// the boundary binary.  An error is not returned if the database is already at
-// the most recent version.
+// the boundary binary. It is not an error for the database to be at the
+// most recent version.
+// An error is returned if the dirty bit has not been set.
```

Comment:
Can you explain this change? Just not following, why would we _want_ the dirty bit to be set after doing migrations?

Reply:
The intent is to fix the behavior of the init command.  Currently if we initialize the schema but fail when setting the default resources the database is considered initialized and not dirty which is not correct IMO.